content/en/core/overview/self-hosting-multiple-nodes.md

```diff
@@ -0,0 +1,285 @@
+---
+title: "Steps for Self Hosting the CHT On Four Nodes (Clustered CouchDB Setup)"
+linkTitle: "Self Hosting - Multiple Nodes"
+weight: 2
+description: >
+  How do you setup the CHT on multiple nodes in a cluster?
+---
+
+### What is a clustered setup?
+
+In a clustered CHT setup, there are multiple couchdb nodes responding to users. This is essential to horizontally scaling your CHT instance. This is one of the capabilities added to the CHT post 4.x release. In this guide, we are going to be setting up a 3 node couchdb cluster. The way the CHT is setup in this version, we require all three couchdb nodes to be up and running. Our healthcheck module determines the health of the couchdb nodes and turns off the CHT if any single node is not functional.
+
+### What are the four nodes used for?
+* Node 1 - CHT-core - runs the core functionality of the CHT like API, sentinel
+* Node 2, 3 & 4 - Couchdb Nodes (A 3 node couchdb cluster)
+
+### Networking Setup
+
+Make sure the following ports are open for the nodes:
+* `7946 TCP` - For docker communication amongst nodes
+* `7946 UDP` - For docker communication amongst nodes
+* `2377 TCP` - Docker cluster management communication
+* `4789 UDP` - Docker overlay network traffic
+* `ICMP` - For ping
+
+**Security Note**: As a security measure, be sure to restrict the IP addresses of the four nodes only to be able to connect to these ports.
+
```

Comment:
Consider adding the setup docker instructions here too.

Reply:
fixed in d929c89

Reply:
oop! This was already there from another commit:

> Provision four Ubuntu servers (22.04 as of this writing) that meet our [hosting requirements](http://localhost:1313/apps/guides/hosting/requirements/) including installing Docker and Docker on all of them.

reverted in dc9f9da